cmd/csi/app/csi.go

```diff
@@ -146,28 +145,36 @@ func handle() {
 		panic(fmt.Sprintf("csi: unable to create controller recover due to error %v", err))
 	}
 
-	ctx := ctrl.SetupSignalHandler()
-	go func() {
-		if err := mgr.Start(ctx); err != nil {
-			panic(fmt.Sprintf("unable to start controller recover due to error %v", err))
+	// Register Fuse Recover
+	if recoverFusePeriod > 0 {
```

Comment:
How about referring this pattern? https://github.com/openkruise/kruise/blob/master/pkg/controller/controllers.go#L62:6